pkg/status/combinedstatus-resolver.go

```diff
@@ -102,6 +103,10 @@ type CombinedStatusResolver interface {
 	//
 	// The returned pointers are expected to be read-only.
 	ResolutionExists(name string) (string, util.ObjectIdentifier, bool)
+
+	// MissingStatusCollectors returns the names of StatusCollector object(s)
+	// that are required by a binding but do not exist.
```

Comment:
Since the caller is relying on it, I would document in the comment here that the returned list is sorted.